### PR TITLE
Changes for adding user-agent in the HEADER of calls

### PR DESCRIPTION
**Pull request by sameerbag**
Changes for adding user-agent in the HEADER of calls

### CODE REVIEW

**Review by kkuzmin — APPROVED** [review]
please bump the version

File: collectors/crowdstrike/utils.js [hunk 1]
```diff
@@ -2,14 +2,16 @@ const RestServiceClient = require('@alertlogic/al-collector-js').RestServiceClie
...
 
 function authenticate(baseUrl, clientId, clientSecret) {
     let restServiceClient = new RestServiceClient(baseUrl);
     return new Promise(function (resolve, reject) {
         return restServiceClient.post(`/oauth2/token`, {
             form: {
                 client_id: clientId,
-                client_secret: clientSecret
+                client_secret: clientSecret,
+                user_agent: USERAGENT
```
[the remark this answers is not in the document]
Comment (sameerbag) [reply collectors/crowdstrike/utils.js:16]: added header with user-agent

File: collectors/crowdstrike/utils.js [hunk 2]
```diff
@@ -25,7 +27,8 @@ function getList(apiDetails, accumulator, apiEndpoint, token) {
         return new Promise(function (resolve, reject) {
             restServiceClient.get(apiDetails.url, {
                 headers: {
-                    'Authorization': `Bearer ${token}`
+                    'Authorization': `Bearer ${token}`,
+                    user_agent: USERAGENT
```
[the remark this answers is not in the document]
Comment (sameerbag) [reply collectors/crowdstrike/utils.js:33]: Added user_agent as per Crowdstrike documentation here https://github.com/CrowdStrike/falconpy/blob/52da908ab8b8c2cfa2585ec07ef28ab6c702a726/src/falconpy/_util.py#L225

File: collectors/crowdstrike/utils.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kkuzmin) [reply collectors/crowdstrike/utils.js:16]: It needs to be 
```
'User-Agent': USERAGENT
```

File: collectors/crowdstrike/utils.js [hunk 3]
```diff
@@ -54,7 +59,8 @@ function getIncidents(ids, apiHostName, token) {
     return new Promise(function (resolve, reject) {
         return restServiceClient.post('', {
             headers: {
-                Authorization: 'Bearer ' + token
+                Authorization: 'Bearer ' + token,
+                user_agent: USERAGENT
```
[the remark this answers is not in the document]
Comment (kkuzmin) [reply collectors/crowdstrike/utils.js:63]: Same below

**Pull request merged**